src/Microsoft.DotNet.Darc/DarcLib/VirtualMonoRepo/RepositoryCloneManager.cs

```diff
@@ -163,15 +169,16 @@ public class RepositoryCloneManager : IRepositoryCloneManager
                 catch
                 {
                     // Ref not found yet, let's try another remote
-                    continue;
-                }
-
-                if (refsToVerify.Count == 0)
-                {
-                    _logger.LogDebug("All requested refs ({refs}) found in {repo}", string.Join(", ", requestedRefs), path);
+                    missingCommit = true;
```

Comment:
Should we log what commit is missing? As we log a message when all commits are found below, it would be useful to also log the complementary information about what's missing.